src/components/Messages/CreateMessageModal/CreateMessageModal.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  IconButton,
+  Typography,
+  Box,
+} from "@mui/material/";
+import CloseIcon from "@mui/icons-material/Close";
+import SearchBarMessages from "./SearchBarMessages";
+import MessagesModalList from "./MessagesModalList";
+import { useState } from "react";
+
+const styles = {
+  dialog: {
+    height: "60%",
+    borderRadius: 4,
```

Comment:
That's weird, I thought this was taken care of in `Theme.tsx`. Can you take a look to see why it doesn't work? Our other dialogs have rounded borders without having to set the border radius, so idk why this one isn't the same

Reply:
I think it has to do with me having the set the height of the dialog through `sx` within `PaperProps` attribute. This seems to override the `Theme.tsx` and if I don't maintain a certain height, it just goes back to minHeight

Reply:
I see, it's fine then